assembly/onpremises-ide-packaging-war-platform-api/src/main/webapp/WEB-INF/classes/codenvy/codenvy-api-configuration.properties

```diff
@@ -56,3 +56,5 @@ che.stacks.images.storage=${catalina.base}/stacks/images
 stop.workspace.scheduler.period=60
 
 machine.ssh.connection_timeout_ms=3000
+
+system_ram.limit.check.scheduler.period=60
```

Comment:
i don't think the property name is good, ram looks like a system resource for me @garagatyi wdyt /